README.md

```diff
@@ -6,9 +6,21 @@
 
 Install dependencies:
 
-`sudo apt-get install cmake libtbb2-dev libaio-dev libsnappy-dev zlib1g-dev libbz2-dev liblz4-dev libzstd-dev librocksdb-dev liblmdb-dev libwiredtiger-dev liburing-dev`
+```
+sudo ./toolbox/pkgs/ubuntu-focal.sh
```

Comment:
This file does not exist
Maybe you can write that there are install helpers for various distributions in toolbox/pkgs?